ui-src/e2e/pages/configuration-PO.ts

```diff
@@ -158,6 +159,7 @@ class ConfigurationPageObject {
     this.rampDelay.sendKeys("e");
   }
   checkNegativeValues() {
+    this.applicationName.sendKeys(1);
```

Comment:
is this a valid test for a text field?

Reply:
This is filling in data. the name of the wrong is misleading and could be better named. As there is no "checking happening." Lets change it to typeNegativeValues